renderdoc/driver/ihv/amd/amd_counters.cpp

```diff
@@ -68,7 +68,7 @@ AMDCounters::AMDCounters(bool dx12DebugLayerEnabled)
 
 bool AMDCounters::Init(ApiType apiType, void *pContext)
 {
-#if DISABLED(RDOC_WIN32) && DISABLED(RDOC_LINUX) && DISABLED(RDOC_GGP)
+#if DISABLED(RDOC_WIN32) && DISABLED(RDOC_LINUX) && DISABLED(RDOC_GGP) && DISABLED(RDOC_ANDROID)
```

Comment:
Please note that this is the one and only change that is not Samsung specific. Without this, though, AMD counters will not work for either Samsung or AMD on Android.